Feature/generate qr

kenz44 opened the pull request.

kenz44 committed:
qr code generated

kenz44 committed:
created end of ride user fragment

kenz44 committed:
Merge branch 'develop' into feature/GenerateQR

kenz44 committed:
Merge branch 'develop' into feature/GenerateQR

kenz44 committed:
Merge branch 'develop' into feature/GenerateQR

kenz44 committed:
Added generateQR to menu

kenz44 committed:
Added comments

kenz44 committed:
Fix comment

kenz44 committed:
Merge branch 'develop' into feature/GenerateQR

rmacgill commented:
Could you add a TextView or something similar that shows what the string used to generate the QR code is?

kenz44 committed:
added ability to see the message stored within the QR code

rmacgill committed:
Edited Menu Title

- Edited menu title to indicate that it's there for testing purposes
- Updated dependencies to resolve small IDE issue
